src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index;
+
     public String getRandomColor() {
-        return null;
+        index = new Random().nextInt(Color.values().length);
```

Comment:
Introduce random as a field in order not to create it each time you call getRadomColor()